src/configure

```diff
@@ -42,35 +57,43 @@ The default configuration is to build and link against static Kaldi libraries.
-  --static-math        Build with static math libraries [default=no]
-  --threaded-math      Build with multi-threaded math libraries [default=no]
-  --threaded-atlas     Build with multi-threaded ATLAS libraries [default=no]
-  --atlas-root=DIR     ATLAS root directory [default=../tools/ATLAS/]
-  --openblas-root=DIR  OpenBLAS root directory
-  --clapack-root=DIR   CLAPACK root directory
-  --mkl-root=DIR       MKL root directory
-  --mkl-libdir=DIR     MKL library directory
-  --mkl-threading=LIB  MKL threading layer [default=sequential]
-                       Supported layers: sequential, iomp, tbb, gomp.
-  --omp-libdir=DIR     OpenMP directory
-  --speex-root=DIR     SPEEX root directory
-  --speex-libdir=DIR   SPEEX library directory
-  --speex-incdir=DIR   SPEEX include directory
-
-Following environment variables can be used to override the compiler
-or to provide additional flags to the compiler/linker.
-  CXX         C++ compiler command
+  --help                Display this help message and exit
+  --version             Display the version of 'configure' and exit
+  --static              Build and link against static libraries [default=no]
+  --shared              Build and link against shared libraries [default=no]
+  --use-cuda            Build with CUDA [default=yes]
+  --cudatk-dir=DIR      CUDA toolkit directory
+  --double-precision    Build with double precision floats [default=no]
+  --static-fst          Build with static OpenFst libraries [default=no]
+  --fst-root=DIR        OpenFst root directory [default=../tools/openfst/]
+  --fst-version=STR     OpenFst version string
+  --mathlib=LIB         Math library [default=ATLAS]
+                        Supported libraries: ATLAS, MKL, CLAPACK, OPENBLAS.
+  --static-math         Build with static math libraries [default=no]
+  --threaded-math       Build with multi-threaded math libraries [default=no]
+  --threaded-atlas      Build with multi-threaded ATLAS libraries [default=no]
+  --atlas-root=DIR      ATLAS root directory [default=../tools/ATLAS/]
+  --openblas-root=DIR   OpenBLAS root directory
+  --clapack-root=DIR    CLAPACK root directory
+  --mkl-root=DIR        MKL root directory
+  --mkl-libdir=DIR      MKL library directory
+  --mkl-threading=LIB   MKL threading layer [default=sequential]
+                        Supported layers: sequential, iomp, tbb, gomp.
+  --omp-libdir=DIR      OpenMP directory
+  --speex-root=DIR      SPEEX root directory
+  --speex-libdir=DIR    SPEEX library directory
+  --speex-incdir=DIR    SPEEX include directory
+  --host=HOST           Host 'triple' for cross compilation
+  --android-incdir=DIR  Andraid include directory
+
+Following environment variables can be used to override the default toolchain
+  CXX         C++ compiler [default=g++]
+  AR          Archive maintenance utility [default=ar]
+  AS          Assembler [default=as]
+  RANLIB      Archive indexing utility [default=ranlib]
```

Comment:
I'll merge this as-is, but a note here to the effect that $HOST will be prepended might be nice.